src/components/Categories/Categories.tsx

```diff
@@ -1,8 +1,177 @@
-import React, { FC } from "react";
+import React, { FC, useState, useEffect } from "react";
+import { categoriesArr, brandArr, starsArr } from "mockedData/mockedData";
+import { calcCategoriesAmount } from "@utils/calcCategoriesAmount";
+import { Checkbox, Rate, Slider, InputNumber } from "antd";
+import { addFilterValues, resetDropDownValues } from "@store/reducers/UserSlice";
+import { IFilterData } from "@models/ICard";
+import { useAppDispatch, useAppSelector } from "@hooks/hooks";
 import s from "./Categories.module.scss";
+import "./checkbox.css";
 
 const Categories: FC = () => {
-  return <div className={s.categories}>Categories</div>;
+  const [reset, setReset] = useState(false);
+  const [input, setInput] = useState<IFilterData>({
+    category: [],
+    brand: [],
+    rating: [],
+    priceMin: 5,
+    priceMax: 100,
```

Comment:
Why are min/max static ?

Reply:
they are not static, this is a starting values ​​after reload

Reply:
You should find min/max price in your products